Scryfall Tools/limited_pools.py

```diff
@@ -4,25 +4,116 @@
 
 
 def get_pack(set_code):
-    # Get a rare or mythic (~14% chance of Mythic)
-    if random.randint(0, 7) == 7:
-        rare_card = st.get_random_card(f'set:{set_code} r=m in:booster')
-    else:
-        rare_card = st.get_random_card(f'set:{set_code} r=r in:booster')
+    def get_cmr_pack(set_code):
+        # Commander Legends pack distribution        
+        pack = []
+        
+        # Get 1 foil
+        # Randomly determine rarity of foil
+        foil_rng = random.randint(0, 119)
+        if(foil_rng == 119):
+            rarity = 'm'
+        elif(foil_rng > 111):
+            rarity = 'r'
+        elif(foil_rng > 88):
+            rarity = 'u'
+        else:
+            rarity = 'c'
```

Comment:
For a weighted choice from a list, you can try the random.choices() method.

Examples: [https://pynative.com/python-weighted-random-choices-with-probability/](url)

```suggestion
        # Get 1 foil
        # Randomly determine rarity of foil
        rarity = random.choices(
            [m', 'r', 'u', 'c'], 
            [1, 8, 23, 88]
        )
```